homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -180,35 +180,35 @@ def available(self) -> bool:
         """Return True if entity is available."""
         return self.entity_description.key == _ONLINE_KEY or self._api.available
 
-    def update(self) -> None:
+    async def async_update(self) -> None:
         """Update entity."""
         if self.entity_description.key == _ONLINE_KEY:
-            self._update_online()
+            await self._update_online()
         else:
-            self._update_others()
+            await self._update_others()
 
     @Throttle(_ONLINE_SCAN_INTERVAL)
-    def _update_online(self) -> None:
+    async def _update_online(self) -> None:
```

Comment:
Since Home Assistant has sync and async code, we prefix async functions with `async_`

Reply:
Sure! Changed these throughout.

Reply:
Sure! Changed these throughout.